src/main/java/com/faforever/api/data/domain/VotingAnswer.java

```diff
@@ -19,6 +20,7 @@
 @Include(type = VotingAnswer.TYPE_NAME)
 @ReadPermission(expression = IsEntityOwner.EXPRESSION)
 @UpdatePermission(expression = "Prefab.Role.None")
+@EqualsAndHashCode(of = {"vote", "votingChoice"}, callSuper = false)
```

Comment:
This could be removed if comment in AbstractEntity is considered. The id alone should be enough.

Reply:
Ok. Accepted :)